runscript.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import unittest
+import xmlrunner
+import inspect
+from module.cliparser import build_parser
+from module.buildcap import build_capabilities
+from selenium import webdriver
+
+# exit codes
+ERROR = 3
+UNSTABLE = 2
+FAILED = 1
+PASSED = 0
+ret = PASSED
+
+parser = build_parser('RWD client for running a script');
+parser.add_argument('--path', required=True)
+
+parser.add_argument('--xml', dest='xml', action='store_true')
+parser.add_argument('--no-xml', dest='xml', action='store_false')
```

Comment:
This is a good point, I'll add help to the commands. Those commands are not in build_parser because the later is used also by singleurl.py which doesn't have a path. At some point, we should get rid of singleurl.py in favor of runscript.py.